docs/UserGuide.md

```diff
@@ -96,6 +96,33 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe e/betsycrowe@example.com a/Newgate Prison p/1234567`
 
+Do take note that there is a near duplicate check for names. This means that when 
+adding a person whose name is similar to another client already found in DonnaFin, 
+the result display will prompt the user as such but the client will still be added to 
+the contact list. However, if the names are complete duplcates they will be rejected.
+
+Near duplicate:
+![DuplicateRejectionMessage](images/NearDuplicateRejectionMessage.png)
+
+Duplicate:
+![FullDuplicateRejectionMessage](./images/FullDuplicateRejectionMessage.png)
+
+Names are said to be near duplicates if the lowercase of both names are the same or when
```

Comment:
How does:
differs only in case (upper case letters where there are lower case letters) or with spaces (user types in two or more spaces in place of one)

sound?

Reply:
noted and amended